src/number.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const { nBN } = require('./bn')
+
+const safeParseInt = (value) => {
+  const num = parseInt(
+    /^0x/.test(value.toString()) ? value : nBN(value).toNumber(),
```

Comment:
i don't think we need support for hex or octal

Reply:
removed the function